mmv1/templates/terraform/examples/apihub_api_hub_instance_basic.tf.tmpl

```diff
@@ -0,0 +1,8 @@
+resource "google_apihub_api_hub_instance" "{{$.PrimaryResourceId}}"{
```

Comment:
You'll need some more tests to check the range of config that can be provided (see https://googlecloudplatform.github.io/magic-modules/test/test/):
- Since this test should be minimal, is `config {}` valid?
- You'll need an "update" test to confirm fields that are mutable can be updated in-place, like description and config
- You'll need to use each field in at least one test (you can do this with the update test, or add an explicit `*_full.tf.tmpl` example with every possible field specified)

Reply:
- config{} wont be valid. One of disable_search or vertex_location would need to be provided. Default value of disable_search would be false. But in that scenario, vertex_location would need to be provided in the config.
- Currently update is also not supported for this resource
- Currently only one resource/project can be provisioned. And since delete and update is not supported for this resource, was not able to add further test in the same flow. Do you have any suggestions for this scenario.

Reply:
I have added a test with all possible fields, which will be provisioned in a separate project. Hope that is fine?

Reply:
For both tests, could you create a separate project for the instance? Otherwise our test environments will become polluted in a way we seemingly cannot clean up. Ie.:
```
resource "google_project" "project" {
  name       = "{{index $.Vars "project_id"}}"
  project_id = "{{index $.Vars "project_id"}}"
  org_id     = "{{index $.TestEnvVars "org_id"}}"
  deletion_policy = "DELETE"
}
```
And in the examples blocks:
```
    vars:
      project_id: 'apihub-proj'
    test_env_vars:
      org_id: 'ORG_ID'
```